### ISSUE TITLE
Ability to filter out some logs

### ISSUE THREAD

**Issue by DavertMik** [logtail/logtail-ruby-rails#22]
Some endpoints like `/`, `/status` are calling regularly and I would like not to log them to logtail
There should be a way to filter out logs for some controllers before sending data

**Comment by PetrHeinz** [comment]
Hi @DavertMik and thank you for the suggestion 🙌 

We are tracking this issue internally. I don't have a time estimate to share at this point, but I will make sure to keep this issue updated.

Thanks for your patience. 🙏

**Comment by PetrHeinz** [comment]
Hello @DavertMik, I've just released [`logtail` gem in `v0.1.12`](https://rubygems.org/gems/logtail/versions/0.1.12) which contains option to filter logs sent to Better Stack.

See `Logtail::Config#filter_sent_to_better_stack`:
```ruby
# This allows filtering logs that are sent to Better Stack. Can be called multiple times, all filters will
# be applied. If the passed block RETURNS TRUE for a particular LogEntry, it WILL NOT BE SENT to Better Stack.
#
# See {Logtail::LogEntry} for available attributes of the block parameter.
#
# @example Rails
#   config.logtail.filter_sent_to_better_stack { |log_entry| log_entry.context_snapshot[:http][:path].start_with?('/_') }
# @example Everything else
#   Logtail.config.filter_sent_to_better_stack { |log_entry| log_entry.message.include?('IGNORE') }
def filter_sent_to_better_stack(&block)
```

I would welcome your feedback 🙂  If you have any questions or issues regarding the new option, don't hesitate to reopen the issue.
